config/locales/de/models.yml

```diff
@@ -76,13 +76,13 @@ de:
         # Countable
         one: "Manager"
         other: "Manager"
-      managed_record: # TODO: Translate
+      managed_record:
         # Basic pluralization
-        single: "Responsibility"
-        plural: "Responsibilities"
+        single: "Verantwortung"
+        plural: "Verantwortlichkeiten"
         # Countable
-        one: "responsibility"
-        other: "responsibilities"
+        one: "Verantwortung"
+        other: "Verantwortlichkeiten"
```

Comment:
These should probably not be capitalised, as they will be used in the middle of a sentence.

Reply:
no, that is correct. welcome to the German Language :)
https://dict.leo.org/englisch-deutsch/responsibility
https://dict.leo.org/englisch-deutsch/responsibilities